### DISCUSSION TITLE
Firebase Realtime

### DISCUSSION THREAD

**Discussion by georgearcego** [atoti/atoti#786] [Q&A] [answered]
Is it possible to connect Atoti with Firebase realtime?

**Comment by tibdex** [comment, marked as answer]
Yes, you could for instance call [`Table.append()`](https://docs.atoti.io/0.7.4/lib/atoti/atoti.table.html#atoti.Table.append) in a [`stream_handler`](https://github.com/thisbejim/Pyrebase/tree/7a652e6bd9d148da5ff6dbe7548c6d5d0dfa1109#streaming).

**Reply by georgearcego** [reply]
Thank you!
